README.md

```diff
@@ -1,5 +1,40 @@
+# GetIN Dashboard Web Client
+
+## Deployment guide to AWS (Linux)
+
+Ensure that you the server pem key file in any of your directories.
+
+Connect to the GetIN instance on aws
+
+### `cd ~/Directory_With_Pem_Key && ssh -i filename.pem ubuntu@server_ip`
```

Comment:
I think we already have the list of server IPs, just give an example for a deployment on production or staging with one of the IPs.